WcaOnRails/app/controllers/api/v0/results_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Api::V0::ResultsController < Api::V0::ApiController
+  def personal_records
+    user = User.find_by_id(params[:user_id])
```

Comment:
Maybe `params.require` if user_id is a mandatory param?

Reply:
Good point! Changed